### PR TITLE
Add the governance wg to team repo

### PR DESCRIPTION
**Pull request by mgattozzi**
cc @skade, @Manishearth, @nikomatsakis, @jonathandturner, and @arshiamufti

I need to add Nell and Benjamin but not sure what their contact info is to put under people

### CODE REVIEW

File: teams/wg-governance.toml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name = "wg-governance"
+subteamof = "core"
```
Comment (Manishearth) [inline teams/wg-governance.toml]: dash

**Comment by Manishearth** [comment]
@nellshamrell @gnunicorn can you both give us a toml file in [this format](https://github.com/rust-lang/team/blob/master/people/Manishearth.toml) with your preferred name/contact information?

**Comment by gnunicorn** [comment]
like this https://github.com/rust-lang/team/pull/48 ?

**Comment by Manishearth** [comment]
Yep, but that can't be merged directly since you need to be on a team for CI to pass :)

**Comment by Manishearth** [comment]
Merged in @gnunicorn's stuff, added Nell, ready to merge

File: people/nellshamrell.toml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+name = "Nell Shamrell-Harrington"
```
Comment (Manishearth) [inline people/nellshamrell.toml]: @nellshamrell lmk if anything here should be changed

**Comment by mgattozzi** [comment]
@Manishearth looks good to me now. We can probably merge

**Pull request merged**